### DISCUSSION TITLE
Question about device specific poll-rate TuYa TS0121_plug

### DISCUSSION THREAD

**Discussion by FlexxFR** [Koenkk/zigbee2mqtt#6337] [Devices]
According zigbee2mqtt documentation this plug can be configured with another poll rate:
https://github.com/Koenkk/zigbee2mqtt.io/blob/develop/docs/devices/TS0121_plug.md#device-type-specific-configuration

`measurement_poll_interval: This device does not support reporting electric measurements so it is polled instead. The default poll interval is 60 seconds.`

I'm trying to set this to 10 seconds but I'm unable to do so. I searched a lot but could not find a working example anywhere.

I tried to add it for "smartplug2" to devices.yaml like so:

```
'0x70a423fffe681234':
  friendly_name: smartplug_1
'0x70a423fffe681235':
  friendly_name: smartplug_2
  measurement_poll_interval: 10
'0x70a423fffe6851235':
  friendly_name: smartplug_3
```

After reboot of HA I did not notice any difference. Is this the right approach or is this device not capable of faster updates?

The documentation states it should be added to the configuration.yaml directly however I'm not sure how as the layout is a bit different. First part:

```
{
  "external_converters": [],
  "devices": [
    "devices.yaml"
  ],
  "groups": [
    "groups.yaml"
  ],
  "homeassistant": true,
  "permit_join": false,
```

I tried to add the lines using HA > Supervisor > Add-on > Zigbee2Mqtt > Configuration (as I assume this is the same file)

devices:
  '0x70a423fffe681235':
    friendly_name: 'smartplug2'
    measurement_poll_interval: 10

But this will give me "Invalid YAML".

**Comment by mabosch74** [comment]
try this one:
`"friendly_name": "smartplug2",
"measurement_poll_interval": 10`

**Reply by FlexxFR** [reply]
Thanks for your help. Tried is with and without quoted but it's not using this settings.

```
'0x60a423fffe6215e3':
  friendly_name: smartplug_dryer, measurement_poll_interval: 30
'0x60a423fffe685185':
  friendly_name: smartplug_washer, measurement_poll_interval: 120
```

After reboot of HA, I use the Zigbee2MQTT logs windows to filter on the zigbee messages from both smartplugs, but they still arrive every 60 seconds. Ignoring both changes values.

BTW, I updated the devices.yaml file in \share\zigbee2mqtt.
